README.md

```diff
@@ -225,30 +259,42 @@ Otherwise the client will return an error `err`:
 </table>
 </details>
 
+<details>
+<summary>Arguments</summary>
 
-### Arguments
-
+#### `emailAddress`
+The email address of the recipient, only required for email notifications.
 
 #### `templateId`
 
 Find by clicking **API info** for the template you want to send.
 
+#### `reference`
+
+An optional identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+
+You can omit this argument if you do not require a reference for the notification.
```

Comment:
Could do that, although having spoken to a node developer he mentioned something about an arguments object rather than just passing in args, so might refactor it all to use the args object pattern instead as it seems like a fairly standard pattern to use for Node development. 

I also wonder if it's worth getting him to review the style of the Node code, not the actual functionality of it, as he would know about best practices as he's in the middle of writing a Node style guide.

Reply:
ok, as refactoring will break existing code, have instead decided to update the README to make it clear to users how to pass in `emailReplyToId` when other optional args are not set